code/datums/uplink_item.dm

```diff
@@ -1046,6 +1047,14 @@ var/list/uplink_items = list()
 	reference = "BRMB"
 	item = /obj/item/clothing/shoes/magboots/syndie
 	cost = 3
+	excludefrom = list(/datum/game_mode/nuclear)
```

Comment:
This means they're a traitor item now; really shouldn't be; if the intent is to disallow nukies access to this, then it should just be removed from the uplink.